frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -5,14 +5,22 @@ action.GitMachete.CheckoutSelectedBranchAction.text=Checkout Selected Branch
 action.GitMachete.CheckoutSelectedBranchAction.GitMacheteContextMenu.text=_Checkout Branch
 action.GitMachete.CheckoutSelectedBranchAction.description=Checkout this branch
 
-action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Rebase Selected Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=_Rebase Branch Onto Parent
-action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Rebase this branch onto parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.text=Checkout and Rebase Selected Branch Onto Parent
+action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.text=Checkout and _Rebase Branch Onto Parent
```

Comment:
:tada:  **Fun fact**  :tada: 
about this `_` within action texts like `Checkout and _Rebase Branch Onto `
(that probably only me wasn't aware of)
___

It is better to see the context - the context menu. 

![image](https://user-images.githubusercontent.com/19799111/81965329-31605600-9618-11ea-8ac6-c9dbcdbb6513.png)

`_` underlines the following letter. It indicates the key shortcut to perform an action within the context menu. 

So in the example presented above:
- pressing **R** - performs `Rollback`
- pressing **D** - performs `Delete`

We should be aware of that to **avoid duplicate definitions**.